### PR TITLE
Dev features

### PR DESCRIPTION
**Pull request by vinnyHero**

### CODE REVIEW

File: app/src/main/java/com/goupwards/FaqExpandableListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import android.widget.TextView;
+
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Vineet on 3/10/2017.
+ */
+
+public class FaqExpandableListAdapter extends BaseExpandableListAdapter {
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/FaqExpandableListAdapter.java]: why you are adding this adapter, It already merged

File: app/src/main/java/com/goupwards/analytics/FCMService.java [hunk 2]
```diff
@@ -18,7 +16,7 @@
   public void onTokenRefresh() {
     // Get updated InstanceID token.
     String refreshedToken = FirebaseInstanceId.getInstance().getToken();
-    Log.d(TAG, "Refreshed token: " + refreshedToken);
+    Log.d("token isssssssssssss", "Refreshed token: is ---->>>>>>>>>>.." + refreshedToken);
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/analytics/FCMService.java]: Please use proper tags , is is "issssss.."

File: app/src/main/java/com/goupwards/login/LoginActivity.java [hunk 3]
```diff
@@ -128,15 +130,15 @@ private void HandleSignRequest(GoogleSignInResult result) {
...
     }
   }
 
+
   private void fetchSMS() {
-    // new algo360 implemented
-//    ExtraHelperFunctions.putRefUserId(context, "~UPWARDS~");
-//    Algo360_SDK_Init.startAlgo360(context);
+
+    ExtraHelperFunctions.putRefUserId(getApplicationContext(), "156");
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/login/LoginActivity.java]: what is 156?

File: app/src/main/res/layout/faq_listgroup.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (manojnagar) [inline app/src/main/res/layout/faq_listgroup.xml]: what is use of this layout?

File: app/src/main/res/layout/faq_listitem.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (manojnagar) [inline app/src/main/res/layout/faq_listitem.xml]: I already created layouts, why duplicate?

File: app/src/main/res/layout/faq_listview_main.xml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+    <ExpandableListView
+        android:id="@+id/lvExp"
```
Comment (manojnagar) [inline app/src/main/res/layout/faq_listview_main.xml]: why adding new layout?

File: app/src/main/java/com/goupwards/analytics/FCMService.java (hunk 2 above)
Comment (vinnyHero) [inline app/src/main/java/com/goupwards/analytics/FCMService.java]: done

File: app/src/main/java/com/goupwards/FaqExpandableListAdapter.java (hunk 1 above)
Comment (vinnyHero) [inline app/src/main/java/com/goupwards/FaqExpandableListAdapter.java]: removed . done

File: app/src/main/java/com/goupwards/login/LoginActivity.java (hunk 3 above)
Comment (vinnyHero) [inline app/src/main/java/com/goupwards/login/LoginActivity.java]: done, changed to customer id

File: app/src/main/res/layout/faq_listgroup.xml (hunk 4 above)
Comment (vinnyHero) [inline app/src/main/res/layout/faq_listgroup.xml]: deleted , done

File: app/src/main/res/layout/faq_listitem.xml (hunk 5 above)
Comment (vinnyHero) [inline app/src/main/res/layout/faq_listitem.xml]: deleted

File: app/src/main/res/layout/faq_listview_main.xml (hunk 6 above)
Comment (vinnyHero) [inline app/src/main/res/layout/faq_listview_main.xml]: deleted , done

**Pull request closed without merging**